ms_test.py

```diff
@@ -0,0 +1,346 @@
+import argparse
```

Comment:
Why do we need a separate multi-scale test?

Reply:
Actually, this can be merged into mask_test.py by adding plenty of  `if and else` clauses, since one more field needs to be reserved for mask scoring (`new_score` in `ms_test.py` file). For clarity, I add one more file to avoid this problem. I haven't thought of other elegant solutions. Or, you mean these `if and else` clauses are just ok?